_unused_topics/images-using-images-s2i-java.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
```

Comment:
Unused content. This is brought forward + modularized in case we need it. Since this content is not building in our docs, and will require additional PRs/reviews I think it is safe to skip it for general review.